Makefile

```diff
@@ -66,6 +66,10 @@ test: export TARGET = dev
 test: ## Run the tests
 	${DOCKER_COMPOSE} run --rm app npm run test
 
+test\:mutation: export TARGET = dev
```

Comment:
Why are these additional explicit exports required. Doesn't the makefile set these up at the top?

Reply:
It forces `TARGET` to be `dev` for `test:mutation` (ie `TARGET=prod make test:mutation` continues to work as `TARGET=prod` is ignored).